fmriprep/cli/run.py

```diff
@@ -108,11 +108,13 @@ def main():
             sentry_sdk.capture_message(success_message, level='info')
 
         # Bother users with the boilerplate only iff the workflow went okay.
-        if (config.execution.output_dir / 'fmriprep' / 'logs' / 'CITATION.md').exists():
+        _boiler_file = config.execution.output_dir / "fmriprep" / "logs" / "CITATION.md"
+        if _boiler_file.exists():
+            _boiler_file = _boiler_file.relative_to(config.execution.output_dir)
```

Comment:
If somebody's using Singularity or manually constructing a Docker command, they'll be able to parse the output directory. What if we condition this on `fmriprep-docker`'s mapping to `/out`?

```suggestion
            if str(config.execution.output_dir) == "/out
                boiler_file = Path("<output_dir>") / boiler_file.relative_to(config.execution.output_dir)
```